Geometry/MTDGeometryBuilder/src/RectangularPixelTopology.cc

```diff
@@ -0,0 +1,353 @@
+#include "Geometry/MTDGeometryBuilder/interface/RectangularPixelTopology.h"
+
+  /**
+   * Topology for rectangular pixel detector with BIG pixels.
+   */
+// Modified for the large pixles.
+// Danek Kotlinski & Michele Pioppi, 3/06.
+// See documentation in the include file.
+
+//--------------------------------------------------------------------
+// PixelTopology interface. 
+// Transform LocalPoint in cm to measurement in pitch units.
+std::pair<float,float>
+RectangularPixelTopology::pixel( const LocalPoint& p ) const
+{
+  // check limits	
+  float py = p.y();
+  float px = p.x();
+  
+#ifdef EDM_ML_DEBUG
```

Comment:
Use LogDebug::log instead of the CPP.

Reply:
See 
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideMessageLogger#Issuing_complex_or_computational

Reply:
Thanks!